charts/consul/templates/api-gateway-controller-deployment.yaml

```diff
@@ -237,4 +237,8 @@ spec:
       nodeSelector:
         {{ tpl .Values.apiGateway.controller.nodeSelector . | indent 8 | trim }}
       {{- end }}
+      { { - if .Values.apiGateway.controller.tolerations } }
```

Comment:
The formatting on the `{{` and `}}` needs correcting here. There is an additional space.